docker/Dockerfile_simulation-jammy

```diff
@@ -0,0 +1,41 @@
+#
+# PX4 Gazebo 7 (Garden) development environment in Ubuntu 22.04 Focal
+#
+
+FROM px4io/px4-dev-base-jammy:latest
+LABEL maintainer="Nuno Marques <nuno.marques@dronesolutions.io>"
```

Comment:
@ThomasDebrunner , don't you want to be the maintainer?

Reply:
Fixed